tests/unit/test_kwargs/test_validations.py

```diff
@@ -46,19 +46,19 @@ def handler(my_key: str = Parameter(**{param_field: "my_key"})) -> None:  # type
 @pytest.mark.parametrize("reserved_kwarg", sorted(RESERVED_KWARGS))
 def test_raises_when_reserved_kwargs_are_misused(reserved_kwarg: str) -> None:
     decorator = post if reserved_kwarg != "socket" else websocket
-
-    exec(f"async def test_fn({reserved_kwarg}: int) -> None: pass")
-    handler_with_path_param = decorator("/{" + reserved_kwarg + ":int}")(locals()["test_fn"])
+    local = dict(locals(), **globals())
```

Comment:
This also caused some weird issues with the scope of the `locals()` output.  This was the only way I could consistently make sure the call to `locals` contained the `test_fn` key